pkg/gci/internal/testdata/already-good.out.go

```diff
@@ -2,7 +2,7 @@ package main
 import (
 	"fmt"
 
-	g "github.com/golang"
-
 	"github.com/daixiang0/gci"
+
+	g "github.com/golang"
```

Comment:
This should not have changed, this looks to be a bug: the gci is running std, default, prefix(github.com/daixiang0/gci).

Reply:
The config order is not the final order, please check README.